fe/fe-core/src/main/java/org/apache/doris/load/loadv2/BulkLoadJob.java

```diff
@@ -324,7 +329,7 @@ public void readFields(DataInput in) throws IOException {
         // The origin stmt does not be analyzed in here.
         // The reason is that it will thrown MetaNotFoundException when the tableId could not be found by tableName.
         // The origin stmt will be analyzed after the replay is completed.
-
+        userInfo = UserIdentity.read(in);
```

Comment:
Modify the `read()` function, you have to update the meta version in `FeMetaVersion.java`

Reply:
> Modify the `read()` function, you have to update the meta version in `FeMetaVersion.java`

done